sections/02_intro.md

```diff
@@ -1,3 +1,25 @@
 ## Introduction
 
-Intro goes here.
+### Potential writing prompt
+
+One potential future that we could imagine is a world in which data, once
```

Comment:
@gwaygenomics : removed them for now. We'll see how the sections come along to decide what to talk about specifically.